### PR TITLE
Feat: Admin Delete, Invite, Edit functionality

### PR DESCRIPTION
**Pull request by claus44**
Resolves: 

## Summary

Implementation of admin features: editing accounts, deleting accounts, and inviting new members

## Acceptance Criteria

- [x] X
- [x] Y

### CODE REVIEW

File: src/views/Home/Home.tsx [hunk 1]
```diff
@@ -92,27 +131,37 @@ function Home(props: Props): React.ReactElement {
...
+  let styleFAB = {
+    right: 16,
+    position: Platform.OS === "web" ? "fixed" : "relative",
+  };
+
+  if (Platform.OS !== "web") {
+    styleFAB["top"] = window.height - safeArea.bottom;
+  } else {
+    styleFAB["bottom"] = 16;
+  }
```
Comment (NickHackman) [inline src/views/Home/Home.tsx]: Oh wait this should match how its done on `main`, see suggestion.

```suggestion
  const styleFAB = {
    position: Platform.OS === "web" ? "fixed" : "absolute",
    margin: 16,
    right: 0,
    bottom: 0,
  };
```

File: src/views/Home/styles.ts [hunk 2]
```diff
@@ -7,12 +9,31 @@ export default StyleSheet.create({
...
   },
 
   list: {
     flex: 1,
   },
+
+  fab: {
+    // TODO: Emulate Fixed for the Floating Action Button
+  },
+
```
Comment (NickHackman) [inline src/views/Home/styles.ts]: remove
```suggestion
```

**Review by NickHackman — CHANGES_REQUESTED** [review]
Sorry! Just these two things, see suggestions for implementing them. You can commit them from Github.

**Review by NickHackman — APPROVED** [review]
LGTM

**Review by freeman91 — APPROVED** [review]

**Pull request merged**